python/tritonserver/__init__.py

```diff
@@ -23,3 +23,61 @@
 # OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""In process Python API for Triton Inference Server."""
+
+import tritonserver._c as _triton_bindings
+from tritonserver._api.wrapper import InferenceRequest as InferenceRequest
+from tritonserver._api.wrapper import InstanceGroupKind as InstanceGroupKind
+from tritonserver._api.wrapper import LogFormat as LogFormat
+from tritonserver._api.wrapper import Metric as Metric
+from tritonserver._api.wrapper import MetricFamily as MetricFamily
+from tritonserver._api.wrapper import MetricFormat as MetricFormat
+from tritonserver._api.wrapper import MetricKind as MetricKind
+from tritonserver._api.wrapper import Model as Model
+from tritonserver._api.wrapper import ModelBatchFlag as ModelBatchFlag
+from tritonserver._api.wrapper import ModelControlMode as ModelControlMode
+from tritonserver._api.wrapper import ModelIndexFlag as ModelIndexFlag
+from tritonserver._api.wrapper import ModelTxnPropertyFlag as ModelTxnPropertyFlag
+from tritonserver._api.wrapper import Options as Options
+from tritonserver._api.wrapper import RateLimitMode as RateLimitMode
+from tritonserver._api.wrapper import Server as Server
+from tritonserver._api._datautils import MemoryAllocator as MemoryAllocator
+from tritonserver._api._datautils import NumpyAllocator as NumpyAllocator
+from tritonserver._c import TRITONSERVER_DataType as DataType
+from tritonserver._c import TRITONSERVER_MemoryType as MemoryType
+
+try:
+    from tritonserver._api._datautils import CupyAllocator as CupyAllocator
+except ImportError:
+    pass
```

Comment:
I see a lot of this `try import cupy; except pass` throughout. I don't think the user will ever actually know if they're falling back to CPU when they could be using GPU had they just installed cupy. 

Can be log warnings somewhere?